dashboard/dashboard-access.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Access TiDB Dashboard
+summary: Learn how to access TiDB Dashboard.
+category: how-to
+---
+
+# Access TiDB Dashboard
+
+To access TiDB Dashboard, visit <http://127.0.0.1:2379/dashboard> via your browser. Replace `127.0.0.1:2379` with the actual PD instance address and port.
+
+> **Note:**
+>
+> If you have deployed multiple PD components in your cluster, you can use the address of only one PD component to access TiDB Dashboard, because the Dashboard operates on only one PD component. If you access via other PD components, your browser is redirected to that PD component. Therefore, if the firewall is not configured for this instance on which PD operates, you might not be able to access TiDB Dashboard.
```

Comment:
Please check [the latest Chinese version](https://docs.pingcap.com/zh/tidb/dev/dashboard-access#%E5%A4%9A-pd-%E5%AE%9E%E4%BE%8B%E8%AE%BF%E9%97%AE) and update it.

Reply:
The updated part contains links of several documents added in https://github.com/pingcap/docs-cn/pull/3440. For convenience, I'll update the change made to `dashboard-access.md` in another PR together with those newly added documents.

Reply:
Got it.